### PR TITLE
BDD Move Content Feature

### PR DESCRIPTION
**Pull request by pcardiga**
BDD scenarios for Move Content feature - https://jira.ez.no/browse/EZP-24187

The current BDD scenarios aim to reflect the behaviour of moving a content in PlatformUI
The situations reflected below are not yet reflected in BDD scenarios due clarification need
- [ ] Try to move an object that is read-only for the user
- [ ] Try to move an object to a read-only location
- [ ] Try to move one object that has one child inaccessible to the user.
- [ ] Try to move into a non-container
- [ ] Copy multiple objects
- [ ] Existence of Manipulate button action view on Action Bar respective sub-bar descendants on right panel
- [ ] Move objects to hidden locations

### CODE REVIEW

**Comment by pcardiga** [comment]
ping @bdunogier , @dpobel , @andrerom

File: Features/CopyMoveDelete/MoveContent.feature [hunk 1]
```diff
@@ -0,0 +1,49 @@
+Feature: Move content using te role of Editor
```
Comment (andrerom) [inline Features/CopyMoveDelete/MoveContent.feature]: the

**Comment by andrerom** [comment]
looks ok, but as mentioned yesterday we should skip scenarios that are currently broken because of API, example hide.

Anyway to mark as skipped?

**Comment by pcardiga** [comment]
The scenarios related with hidden objects are going to be removed for now and included later when the feature is fixed.
In Behat 2.x, there was the possibility of having a tag, such as @skipped, and then configure behat.yml for the scenarios that have that tag be ignored during execution. But for some reason, this possibility is no longer present in Behat 3.0.

**Comment by bdunogier** [comment]
According to the behat docs, tags should still be supported (at least from
the config file):
http://docs.behat.org/en/latest/guides/5.suites.html#suite-filters.

On Tue, Jun 16, 2015 at 11:29 AM, Paulo Cardiga <notifications@github.com>
wrote:

> The scenarios related with hidden objects are going to be removed for now
> and included later when the feature is fixed.
> In Behat 2.x, there was the possibility of having a tag, such as @skipped
> <https://github.com/skipped>, and then configure behat.yml for the
> scenarios that have that tag be ignored during execution. But for some
> reason, this possibility is no longer present in Behat 3.0.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ezsystems/PlatformUIBundle/pull/240#issuecomment-112362432>
> .
>



-- 
--
Bertrand Dunogier
Lead Engineer eZ extensibility
eZ Systems AS | skype:bertrand-dunogier | http://www.ez.no/

**Comment by miguelcleverti** [comment]
I tried using the filters configuration in the behat.yml and it was telling me that the parameter was not recognized, I made some investigations and and found that it might have to do with the behat version. Iit is still possible to use it on the console command such as `php behat  (...) --tags '~@skipped'` which will run everything but the ones with the tag `@skipped`. I will try to investigate a bit more about this anyway.

**Comment by pcardiga** [comment]
Scenarios updated:
ping @bdunogier, @dpobel, @andrerom

**Comment by pcardiga** [comment]
Scenario sentences updated accordingly dicussion occurred in [Copy BDD PR](https://github.com/ezsystems/PlatformUIBundle/pull/243/ "PR for the BDD Copy feature")

**Comment by pcardiga** [comment]
Scenario sentences updated again accordingly new dicussion occurred in [Copy BDD PR](https://github.com/ezsystems/PlatformUIBundle/pull/243/ "PR for the BDD Copy feature")

**Comment by andrerom** [comment]
+1, ref https://github.com/ezsystems/PlatformUIBundle/pull/243#issuecomment-119960457

**Comment by bdunogier** [comment]
+1

**Pull request merged**